src/platform/f4xx/stm32f446xx.cpp

```diff
@@ -0,0 +1,75 @@
+#include <HALf4/stm32f4xx.h>
+#include <HALf4/stm32f4xx_hal.h>
+#include <HALf4/stm32f4xx_it.h>
+
+#include <EVT/platform/f4xx/stm32f4xx.hpp>
+#include <iostream>
+
+using namespace std;
+namespace EVT::core::platform {
+
+void stm32f4xx_init() {
+    HAL_Init();
+    RCC_OscInitTypeDef RCC_OscInitStruct = {0};
+    RCC_ClkInitTypeDef RCC_ClkInitStruct = {0};
+    RCC_PeriphCLKInitTypeDef PeriphClkInit = {0};
+
+    /** Configure the main internal regulator output voltage
+    */
+    __HAL_RCC_PWR_CLK_ENABLE();
```

Comment:
I'm pretty sure you're not supposed to call macros that start with `__`. Can you try to find a wrapper function or something to do this instead? Where did you find this code?

Reply:
Did you find a resolution for this?

Reply:
no, i dont want to touch code that was already there and works

Reply:
although it is interesting that the comment above is wrong